openassessment/xblock/static/sass/oa/utilities/_developer.scss

```diff
@@ -101,6 +101,8 @@ $link-hover: $edx-blue-l1 !default; // from our Pattern Library http://ux.edx.or
 
     input {
       display: block;
+      padding-top: 0;
```

Comment:
Note to reviewers: this very targeted fix is to address TNL-4021 (a special gift for @robrap). The cause of the issue is some general styling in the "xb" directory for all inputs. The padding is set to 10 px (in all dimensions), causing the text to be cut off slightly on Firefox.

I don't want to spend long on this, and I was afraid of introducing other unknown issues if I changed the general rule. However, if we don't feel comfortable with this workaround, I'm happy to remove the commit from the PR.

As far as the "xb" directory, this is what @talbs says: "the xb stuff was a set of styling that I abstracted in case it was useful to other xblocks that would use Sass to compile their CSS
so its mostly general UI and base stlying
I don't think anyone ever documented that or leveraged it, so feel free to merge/combine".